Adding basic examples to node (#903)
OctopusDeploy/docs: docs/octopus-rest-api/octopus.server.exe-command-line/node.md

      --help                 Show detailed help for this command
```

## Basic examples

> This example will drain the default node

Reviewer: ```suggestion

This example drains the default node:
```

      --help                 Show detailed help for this command
```

## Basic examples

This example drains the default node:
```
octopus.server node --drain="true"
```

> This example sets the task cap for the node to 15 on instanace `MyNewInstance`:

Reviewer: ```suggestion
This example sets the task cap for the node to 15 on instance `MyNewInstance`:
```

> This example will set the task cap for the node to 15 on instanace MyNewInstance

Reviewer: ```suggestion
This example sets the task cap for the node to 15 on instanace `MyNewInstance`:
```
